itl/plugins-contrib.d/logfiles.conf

```diff
@@ -0,0 +1,115 @@
+/*
+ * Icinga2 CheckCommand definitions to monitor log files using check_logfiles.
+ * The check_logfiles is a Plugin for Icinga which scans log files for specific patterns.
+ * Available from here: https://labs.consol.de/nagios/check_logfiles/
+ */
+
+object CheckCommand "log" {
+	import "ipv4-or-ipv6"
+
+	command = [ PluginContribDir + "/check_logfiles.pl" ]
```

Comment:
AFAIK the plugin is installed without the `.pl` suffix, configure generates this script like this.